libbeat/docs/dashboardsconfig.asciidoc

```diff
@@ -10,7 +10,12 @@
 //////////////////////////////////////////////////////////////////////////
 
 [[configuration-dashboards]]
+ifndef::deprecate_dashboard_loading[]
 == Load the Kibana dashboards
+endif::[]
+ifdef::deprecate_dashboard_loading[]
+== Load the Kibana dashboards (deprecated)
```

Comment:
I still wonder if you really want to have this in the title instead of a deprecate box below as we do normally?